freqtrade/rpc/discord.py

```diff
@@ -0,0 +1,108 @@
+import json
+import logging
+from typing import Dict, Any
+
+import requests
+
+from freqtrade.enums import RPCMessageType
+from freqtrade.rpc import RPCHandler, RPC
+
+
+class Discord(RPCHandler):
+    def __init__(self, rpc: 'RPC', config: Dict[str, Any]):
+        super().__init__(rpc, config)
+        self.logger = logging.getLogger(__name__)
+        self.strategy = config.get('strategy', '')
+        self.timeframe = config.get('timeframe', '')
+        self.config = config
+
+    def send_msg(self, msg: Dict[str, str]) -> None:
+        self._send_msg(msg)
+
+    def _send_msg(self, msg):
+        """
+        msg = {
+            'type': (RPCMessageType.EXIT_FILL if fill
+                     else RPCMessageType.EXIT),
+            'trade_id': trade.id,
+            'exchange': trade.exchange.capitalize(),
+            'pair': trade.pair,
+            'leverage': trade.leverage,
+            'direction': 'Short' if trade.is_short else 'Long',
+            'gain': gain,
+            'limit': profit_rate,
+            'order_type': order_type,
+            'amount': trade.amount,
+            'open_rate': trade.open_rate,
+            'close_rate': trade.close_rate,
+            'current_rate': current_rate,
+            'profit_amount': profit_trade,
+            'profit_ratio': profit_ratio,
+            'buy_tag': trade.enter_tag,
+            'enter_tag': trade.enter_tag,
+            'sell_reason': trade.exit_reason,  # Deprecated
+            'exit_reason': trade.exit_reason,
+            'open_date': trade.open_date,
+            'close_date': trade.close_date or datetime.utcnow(),
+            'stake_currency': self.config['stake_currency'],
+            'fiat_currency': self.config.get('fiat_display_currency', None),
+        }
+        """
+        self.logger.info(f"Sending discord message: {msg}")
+
+        # TODO: handle other message types
+        if msg['type'] == RPCMessageType.EXIT_FILL:
+            profit_ratio = msg.get('profit_ratio')
+            open_date = msg.get('open_date').strftime('%Y-%m-%d %H:%M:%S')
+            close_date = msg.get('close_date').strftime(
+                '%Y-%m-%d %H:%M:%S') if msg.get('close_date') else ''
```

Comment:
Please use `DATETIME_PRINT_FORMAT` (imported from `freqtrade.constants`). It's the same dateformat - but it's not "everywhere" in the code.